src/components/calcite-input-date-picker/readme.md

```diff
@@ -8,15 +8,9 @@
 
 ```html
 <div style="width: 400px">
-  <calcite-label
-    layout="inline"
-    class="sc-calcite-label-h sc-calcite-label-s"
-    alignment="start"
-    status="idle"
-    scale="m"
-  >
+  <calcite-label layout="inline" alignment="start" status="idle" scale="m">
     <!---->
-    <label class="sc-calcite-label sc-calcite-label-s">
+    <label>
```

Comment:
This is the only change we want right? (The rest seems like another instance of #2433.)

Reply:
Oh wait, it looks like we need to remove the internal native label from this example, replace with something like this:

```html
      <calcite-label layout="inline">
        Date
        <calcite-input-date-picker
          min="2016-08-09"
          max="2023-12-18"
          intl-next-month="Next month"
          intl-prev-month="Previous month"
          layout="horizontal"
        ></calcite-input-date-picker>
      </calcite-label>
```

Maybe this pr can be scrapped then?

Reply:
@caripizza, I'll put in a PR to fix that rn

Reply:
Thanks @benelan!